docs/architecture/microservices/multi-container-microservice-net-applications/subscribe-events.md

```diff
@@ -100,7 +100,7 @@ As mentioned earlier in the architecture section, you can have several approache
 
 - Using [transaction log mining](https://www.scoop.it/t/sql-server-transaction-log-mining).
 
-- Using the [Outbox pattern](http://gistlabs.com/2014/05/the-outbox/). This is a transactional table to store the integration events (extending the local transaction).
+- Using the [Outbox pattern](http://www.kamilgrzybek.com/design/the-outbox-pattern/). This is a transactional table to store the integration events (extending the local transaction).
```

Comment:
Just tested and this could be an https link. Sorry for the additional comment.

```suggestion
- Using the [Outbox pattern](https://www.kamilgrzybek.com/design/the-outbox-pattern/). This is a transactional table to store the integration events (extending the local transaction).
```